dlm/llm.go

```diff
@@ -304,12 +333,24 @@ func (l *RWLockStruct) unlock() (err error) {
 			panic("track.owners < 0!!!")
 		}
 	}
+	// record the release of the lock
+	track.rwMutexTrack.DLMUnlockTrack(track)
 
 	// See if any locks can be granted
 	processLocalQ(track)
 
 	track.Mutex.Unlock()
 
+	// can't return the
+	if deleted {
+		if track.waitReqQ.Len() != 0 || track.waiters != 0 || track.state != stale {
+			panic(fmt.Sprintf(
+				"localLockTrack object %p retrieved from pool does not have an empty waitReqQ",
```

Comment:
Probably does not matter but this message should be unique and not conflict with error message at line 248 above